contracts/staking/stakeManager/StakeManager.sol

```diff
@@ -114,8 +118,10 @@ contract StakeManager is IStakeManager, StakeManagerStorage {
     }
 
     function startAuction(uint256 validatorId, uint256 amount) external onlyWhenUnlocked {
+        uint256 currentValidatorAmount = validators[validatorId].amount;
```

Comment:
can be made memory?

Reply:
Do you mean load validators[validatorId] whole struct into memory?

Reply:
just the currentValidatorAmount var